test/e2e/operator_test.go

```diff
@@ -2632,6 +2638,9 @@ func newHostNetworkController(name types.NamespacedName, domain string) *operato
 			EndpointPublishingStrategy: &operatorv1.EndpointPublishingStrategy{
 				Type: operatorv1.HostNetworkStrategyType,
 			},
+			NodePlacement: operatorv1.NodePlacement{
+				NodeSelector: &selector,
```

Comment:
Why is this needed?

Reply:
I was trying to fix some tests that were failing due to unrelated changes in API. I will remove the commit.